docs/content/v2.6/quick-start/install/linux.md

```diff
@@ -94,13 +94,13 @@ Starting from Ubuntu 20.04, `python` isn't available anymore. An easy fix is to
 1. Download the YugabyteDB package using the following `wget` command.
 
     ```sh
-    $ wget https://downloads.yugabyte.com/yugabyte-2.6.5.0-linux.tar.gz
+    $ wget https://downloads.yugabyte.com/yugabyte-2.6.6.0-linux.tar.gz
     ```
 
 1. Extract the package and then change directories to the YugabyteDB home.
 
     ```sh
-    $ tar xvfz yugabyte-2.6.5.0-linux.tar.gz && cd yugabyte-2.6.4.0/
+    $ tar xvfz yugabyte-2.6.6.0-linux.tar.gz && cd yugabyte-2.6.4.0/
```

Comment:
```suggestion
    $ tar xvfz yugabyte-2.6.6.0-linux.tar.gz && cd yugabyte-2.6.6.0/
```

Reply:
Modify the yugabyte version for the other command too. Suggested this change even for the macos page.